assets/src/design-system/components/switch/index.js

```diff
@@ -65,7 +96,10 @@ const HiddenRadioButton = styled.input.attrs({ type: 'radio' })`
   overflow: hidden;
 `;
 
-const RadioButtonLabel = styled(Text).attrs({ forwardedAs: 'label' })`
+const RadioButtonLabel = styled(Text).attrs({
```

Comment:
I cannot click anywhere inside the label, even though it looks like. I have to click on the text for it to register. This appears counter-intuitive:

![switch-bug](https://user-images.githubusercontent.com/637548/111839186-c7b47680-88d0-11eb-8f29-8b48589be1d9.gif)

Reply:
Yeah good call, I can fix that!

Reply:
Just fixed that. The label now covers the whole height, not just the text height